### ISSUE TITLE
using symlinks instead of copies possible?

### ISSUE THREAD

**Issue by stanstrup** [sjewo/shiny-loadbalancer#2]
Hi.

Would it be possible to use symlinks instead of copies? that would be more convenient.
It seems lsof can only return the target of the symlink and not the location of the symlink itself so it might be tricky.

**Comment by sjewo** [comment]
I don't think that this is an option with lsof. Maybe you could use the shiny-server logs to monitor the actual connections.
